calcom/cal.com
# [CAL-1008] Fix 'All Bookings' filter not getting selected after coming from 'Your Bookings'

## What does this PR do?

[CAL-1008] Fix 'All Bookings' filter not getting selected after coming from 'Your Bookings'

Fixes #6937 

https://github.com/calcom/cal.com/assets/37188392/f7dd447f-cccd-42a1-a94d-3c96d767fc45


**Environment**: Staging(main branch) / Production

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

- Navigate to `/bookings/upcoming`
- Check the `All Bookings` filter
